SearchingPrivateCandidates.md

```diff
@@ -0,0 +1,9 @@
+
+In order to search you will need to know a little about the customer. In mysupply the customer matches the clientkey. A clientkey is a mysupply wrapper for an accountdid. To get the clientkey for an accountdid [you will want to use the Client API](https://github.com/careerbuilder/CandidateDataServicesDocumentation/blob/master/ClientAPI.md#get-1)
+ 
+To see the makeup of the customer do a [GET on the customer API](https://github.com/careerbuilder/CandidateDataServicesDocumentation/blob/master/CustomerAPI.md) where customer=clientkey.
+ 
+With the customer information you then construct a [search request](http://careerbuildersearch.com/docs/candidate). The key is to limit the values and facets you request to only those that you need. This example demonstrates what a more complex search looks like.
```

Comment:
beef comment - I wouldn't say the key is to limit the values and facets you need, but we recommend that.